lib/api_auth/request_drivers/rest_client.rb

```diff
@@ -29,13 +29,13 @@ def calculated_md5
       end
 
       def populate_content_md5
-        return unless %i[post put].include?(@request.method)
+        return unless %i[post put].include?(@request.method.to_sym)
```

Comment:
Since RestClient returns a string now (and probably moving forward), maybe it would make sense to change the `%i[post put]` to `%w[post put]` and maybe have `@request.method.to_s` for backwards compatibility

Reply:
If you prefer, done !